opnilog-inference-service/start_opnilog_inference.py

```diff
@@ -77,13 +85,17 @@ async def infer_logs(logs_queue):
     saved_preds = defaultdict(float)
     load_cached_preds(saved_preds)
     opnilog_predictor = OpniLogPredictor()
-    if IS_CONTROL_PLANE_SERVICE or IS_RANCHER_SERVICE:
+    if IS_CONTROL_PLANE_SERVICE or IS_RANCHER_SERVICE or IS_LONGHORN_SERVICE:
```

Comment:
all these if condition with 3 ors should be replaced by if service type is not workload.